__tests__/main.test.ts

```diff
@@ -280,6 +280,20 @@ describe("VIB", () => {
       expect(pipeline).not.toEqual("")
     })
 
+    it("Create not default pipeline with parameter with a file", async () => {
+      process.env["INPUT_PIPELINE"] = "vib-pipeline-file.json"
+      process.env["INPUT_RUNTIME-PARAMETERS-FILE"] = "runtime-parameters-file.yaml"
+      const config = await loadConfig()
+      expect(config.runtimeParametersFile).not.toEqual("")
+    })
+
+    it("Error to create not default pipeline without parameter with a file", async () => {
```

Comment:
There is really no error expected here. The test name is misleading.

Reply:
I did for debugging changes, I just removed them.